
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      <link rel="icon" href="../img/idec-wiki-logo.png">
      <meta name="generator" content="mkdocs-1.4.0, mkdocs-material-8.5.6">
    
    
      
        <title>Method - iDEC 2022 | LZU-CHINA</title>
      
    
    
      <link rel="stylesheet" href="../assets/stylesheets/main.20d9efc8.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.cbb835fc.min.css">
        
          
          
          <meta name="theme-color" content="#02a6f2">
        
      
      

    
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="" data-md-color-primary="light-blue" data-md-color-accent="blue">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#molecular-biology" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="iDEC 2022 | LZU-CHINA" class="md-header__button md-logo" aria-label="iDEC 2022 | LZU-CHINA" data-md-component="logo">
      
  <img src="../img/logo-small.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            iDEC 2022 | LZU-CHINA
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Method
            
          </span>
        </div>
      </div>
    </div>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/idec-teams/2022_LZU-CHINA" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


<nav class="md-nav md-nav--primary" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="iDEC 2022 | LZU-CHINA" class="md-nav__button md-logo" aria-label="iDEC 2022 | LZU-CHINA" data-md-component="logo">
      
  <img src="../img/logo-small.png" alt="logo">

    </a>
    iDEC 2022 | LZU-CHINA
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/idec-teams/2022_LZU-CHINA" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        Home
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../description/" class="md-nav__link">
        Description
      </a>
    </li>
  

    
      
      
      

  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" data-md-toggle="toc" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          Method
          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        Method
      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#molecular-biology" class="md-nav__link">
    Molecular Biology
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#characterization-and-analysis-of-reporting-strains-in-vitro" class="md-nav__link">
    Characterization and Analysis of Reporting Strains in Vitro
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cell-culture" class="md-nav__link">
    Cell culture
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#utility-test-for-cell-culture-samples" class="md-nav__link">
    Utility test for cell culture samples
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#co-culture-of-ae-and-le-strains-and-colorectal-cancer-cells-in-vitro" class="md-nav__link">
    Co-Culture of AE and LE Strains and Colorectal Cancer Cells in Vitro
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bacterial-adhesion-analysis" class="md-nav__link">
    Bacterial adhesion analysis
  </a>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../result/" class="md-nav__link">
        Result
      </a>
    </li>
  

    
      
      
      

  
  
  
    <li class="md-nav__item">
      <a href="../safety/" class="md-nav__link">
        Safety
      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#molecular-biology" class="md-nav__link">
    Molecular Biology
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#characterization-and-analysis-of-reporting-strains-in-vitro" class="md-nav__link">
    Characterization and Analysis of Reporting Strains in Vitro
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#cell-culture" class="md-nav__link">
    Cell culture
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#utility-test-for-cell-culture-samples" class="md-nav__link">
    Utility test for cell culture samples
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#co-culture-of-ae-and-le-strains-and-colorectal-cancer-cells-in-vitro" class="md-nav__link">
    Co-Culture of AE and LE Strains and Colorectal Cancer Cells in Vitro
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#bacterial-adhesion-analysis" class="md-nav__link">
    Bacterial adhesion analysis
  </a>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  <a href="https://github.com/idec-teams/2022_LZU-CHINA/blob/main/docs/method.md" title="Edit this page" class="md-content__button md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"/></svg>
  </a>


  <h1>Method</h1>

<h2 id="molecular-biology">Molecular Biology</h2>
<p>We obtain information on lldPRD operon, pCaCd, pPepT, YebF, HlyE, Bxb1, TP901, Amplifying genetic switches, CCL21, CDD-iRGD, φX174 E, INP and HlpA from NCBI, iGEM and BioCyc databases, and amplify using colony PCR from E. coli DH5α, or synthesize (Genewiz, China). mRFPs were deposited and provided by our laboratory. We purchase E. coli Nissle 1917 (EcN) from Biobw (China, Beijing). E. coli DH5α were deposited from laboratory. All plasmids base on the pSB1A3/pSB3T5/pSB4C5 plasmid backbone. pSB1A3 is ampicillin resistant and carries the highcopy replicon ColE1; pSB3T5 is tetracycline resistant and carries the medium copy replicon p15A; pSB4C5 is chloramphenicol resistant and carries the low copy replicon pSC101. All Biobrick parts will be registered in the iGEM Standard Bioparts Registry (partsregistry.org). All amplification primers were synthesized by Tsingke (China, Beijing).</p>
<p>We use a one-step cloning kit (C113, Vazyme, China) to assemble recombinant plasmids (constructions), including Reporter 01-03; Controller 01-06; Switch 01-06; Effector 01-03 (full HlpAmstrain information is provided in Supplementary Materials). Both Reporter and Controller use pSB1A3 as the plasmid backbone. Switch uses pSB4C5 as the plasmid backbone and Effector uses pSB3T5 as the plasmid backbone. All recombinant plasmids were first transformed into E. coli DH5α for amplification and proofed by DNA sequencing (GENERAL Biol,Chuzhou, China), then transformed into EcN.</p>
<p>Reporter consists of an operational unit that senses hypoxia/pH/L-lactate and an mRFP, which purpose was to verify the in vitro response of pPepT/pCaCd/lldPRD operon by fluorescent protein mRFP. Controller introduce the genes TP901 and Bxb1 encoding serine integrase to the downstream of pPepT/pCaCd/lldPRD operon to control the Amplifying genetic switches. Switch 01- 02 were composed of Amplifying genetic switches (TP901 or Bxb1 switches) and fluorescent protein mRFP; Switch 03-04 were composed of Amplifying genetic switches and lysis gene φ X174 E; Switch 05-06 were composed of Amplifying genetic switches and YebF -HlyE composition. Finally, Effector controls the expression of therapeutic proteins HlyE, CCL21 or CDD-iRGD by the constitutive promoter J23100.</p>
<p>We transform Controller and Switch 01/02 into EcN (Amplifying reporter, AR 01-06) to check the signal amplification effect of the Amplifying genetic logic gate, and compare with the Reporter to determine the best induction-response system. We transform the Controller and Switch 05/06 verified in the previous step into EcN (Amplifying Effector, AE) to endow it with the ability to eliminate colorectal cancer. Finally, we construct LE strain (Lysis Effector), which consisted of Controller 05/06, Switch 03/04 and Effector 01-03. To check the lysis ability of φX174 E, we replace the Effector (Lysis reporter, LR) by pSB3T5 (Reporter variant, RV) constitutively expressing mRFP. Since Controller, Switch and Effector possess different replicons and resistances, that ensure the plasmid compatibility.</p>
<h2 id="characterization-and-analysis-of-reporting-strains-in-vitro">Characterization and Analysis of Reporting Strains in Vitro</h2>
<p>We culture each strain in M9 basic medium with relevant antibiotics (supplemented with 0.4% glucose) overnight (37℃, 150 rpm), followed for the further experiment next morning. For lactic acid reporters (Reporter 01; AR 01/02; LR), the culture of each lactic acid reporter variant strain was transferred to a 24 well plates (6 repeats), with M9 broth diluted to 1 mL per well. The next step, the final concentration of L-lactic acid should reach 0, 0.1, 1,5 and 10 mM, by adding 25μL M9 broth containing different concentrations of L-lactic acid (Sigma-Aldrich). For pH reporter (Reporter 02, AR 03/04, LR), each variety strain needs to be cultured in the 96 well plates, and the pH value of M9 medium should be 5.5, 5.8, 6.3 and 7.3, respectively. For hypoxia reporters (Reporter 03, AR 05/06, LR), we culture each variant strain overnight under normoxic and hypoxic conditions at 37℃. To achieve hypoxia, reporters were incubated statically in an anaerobic pocket(Hopebiol, Qingdao, China) with oxygen indicators. We select untransformed EcN and EcN containing p23114-mRFP which grows under the same conditions as negative control and positive control respectively.</p>
<p>The initial OD600 value of all strains is 0.1. After 16-20 hours’ incubation, we detect OD600 and mRFP fluorescence through microplate reader (Thermo Fisher Scientific, USA) (excitation λ: 584nm/10 nm, emission λ: 607/10nm). The fluorescence of all reporters in vitro is calculated by dividing the crude mRFP pixel intensity by OD600, and both signals we obtain from the data of microplate reader. In addition, the background fluorescence signal need to be deducted (no plasmid EcN control). For hypoxia reporters, the fluorescence signal was normalized through the control of constitutive promoter (p23114) to explain the protein folding maturation under hypoxia.</p>
<p>The fold change of each reporter was quantified as the ratio of normalized fluorescence signal under induced and non-induced conditions. We average over all three duplicate values. The baseline conditions are 20% oxygen concentration, 0 mM lactic acid concentration, pH <span class="arithmatex">\(7.3^{25}\)</span>.</p>
<h2 id="cell-culture">Cell culture</h2>
<p>We purchase CT-26 (murine colon cancer cells), RKO (human colon cancer cells) and SW480 (human colon cancer cells bacteria) cells from Procell (Wuhan, China). Cells were all cultured in 25 cm2 culture flasks (Corning, USA) at 37 °C with 5% CO2. The basic culture medium is DMEM (BasalMedia , Shanghai, China), supplemented with 10% (v/v) fetal bovine serum (Gemini, USA) and 1% P/ S (Gibco, USA).</p>
<h2 id="utility-test-for-cell-culture-samples">Utility test for cell culture samples</h2>
<p>With an initial cell count of 105, cell lines Were all seeded in 25cm2 culture flask (11 repeats) containing 5ml DMEM/10% fetal bovine serum. For the next 5 days, we take out the entire contents
of each bottle (two bottle of samples taken at the last time) and transfer to the 15ml sterile centrifuge tube every 8-12 hours. Centrifuge the samples for 5 minutes in 200 r.c.f., collect the supernatant from the culture medium to another centrifuge tube, freezing at -80°C. Subsequently, we inoculate 6 reporter strains (3 of selected pH, lactic acid and hypoxia reporter, cleavage reporter LR 01-03), negative control and positive control in a 24 well plates with 800 μL M9 medium (0.4% glucose) per well. The cell supernatant preserved at 200 μL was co-incubated with the above strains (the volume ratio of cell supernatant to M9 medium was 1:4). In the culture room connected to the CO2-O2 controller, hypoxia biosensors were tested with different levels of oxygen. All strains were cultured for 12 to 16 hours, and we measure the pixel intensity and absorbance of the reporters through microplate reader (Thermo Fisher Scientific, USA). The content of lactic acid and pH in the cell supernatant was calculated through the standard curve of the reporter strains<sup id="fnref:1"><a class="footnote-ref" href="#fn:1">1</a></sup>.</p>
<h2 id="co-culture-of-ae-and-le-strains-and-colorectal-cancer-cells-in-vitro">Co-Culture of AE and LE Strains and Colorectal Cancer Cells in Vitro</h2>
<p>RKO cells were inoculated in 96 well plates (42 repeats) which containing 100 μl DMEM/10% fetal bovine serum cultured fluid (without phenol red). When RKO cells adherent grow, inoculating 10 μl
of AR and LR strains (6 strains in total, 3 repeats, initial OD600 = 0.6) characterize the expression of lysis genes and bacterial growth density during co-culture of engineered AE strains with RKO cells. A CO2 incubator is used to adjust oxygen levels. Fluorescence intensity changes of the reporter were measured by a microplate reader (Thermo Fisher Scientific, USA) (every 20 min). We inoculate 10 μl of AE strains (Activators+ Effector 01, 3 strains in total, 3 repeats), control 1 (Effector 01, 1 strain, 3 repeats), control 2 (Activators, 3 strains, 3 repeats) and negative control (EcN, 1 strain, 3 repeats). Using the CCK8 kit (C0037, Beyotime, China) monitor cell viability changes (every 20 min)29. To investigate the effect of LE strain lysates on tumor cells, LE strains and their controls (lysis+/-, activators +/-) were induced overnight using cell supernatants on day 5 as described in method 4. Centrifugation at 10,000 rpm to collect the bacterial medium supernatant and preserved at -80°C. Subsequently, the cell lines were inoculated in 24 well plates containing 100 μl DMEM/10% fetal bovine serum cultured fluid (24 repeats). After the cells adhered to the wells, we add 10 μl of bacterial medium supernatant to the cell culture medium. Using the CCK8 kit detect cell viability after 1h incubation with bacterial lysates<sup id="fnref:2"><a class="footnote-ref" href="#fn:2">2</a></sup>.</p>
<h2 id="bacterial-adhesion-analysis">Bacterial adhesion analysis</h2>
<p>We add cell slides in 6 cm Petri dishes (Corning, USA), and cultivate CT26/RKO/SW480 cells 2 days in advance. Bacteria expressing INP-HlpA were activated with 5 mL test tubes a day in advance. Through centrifugation collect 5 mL of bacteria and resuspend with 2 mL of PBS. The PBS suspension was submerged in a 6 cm cell culture dish containing cell slides. After 1 h incubation at 37°C, discard the suspension and wash 3 times with PBS. Add 1 ml of DAPI working fluid (KeyGEN biotech, Jiangsu, China) to the culture dish, light avoidance incubation for 10 min, wash once with PBS, add buffer A dropwise on the microscope slide, and flip the cell slide onto the microscope slide. we observe the cell slides by using a fluorescence microscope Axio Imager. Z2 (ZEISS, Germany) with an excitation wavelength of 561 nm (corresponding to microscope AF546 wavelength), and the images were processed using imageJ 2.1.4.7.28. To quantify the amount of adherent bacteria, we dilute 10-7 for the washed down PBS suspension and apply it to LB plates for CFU counting<sup id="fnref:3"><a class="footnote-ref" href="#fn:3">3</a></sup>.</p>
<div class="footnote">
<hr />
<ol>
<li id="fn:1">
<p>Chien, T., et al. Enhancing the tropism of bacteria via genetically programmed biosensors. Nat Biomed Eng 6, 94-104 (2022). https://doi.org/10.1038/s41551-021-00772-3&#160;<a class="footnote-backref" href="#fnref:1" title="Jump back to footnote 1 in the text">&#8617;</a></p>
</li>
<li id="fn:2">
<p>Din, M.O., et al. Synchronized cycles of bacterial lysis for in vivo delivery. Nature 536, 81-85 (2016). https://doi.org/10.1038/nature18930&#160;<a class="footnote-backref" href="#fnref:2" title="Jump back to footnote 2 in the text">&#8617;</a></p>
</li>
<li id="fn:3">
<p>Akman, P.K., Ozulku, G., Tornuk, F. &amp; Yetim, H. Potential probiotic lactic acid bacteria isolated from fermented gilaburu and shalgam beverages. LWT- Food Science and Technology 149, 111705 (2021). https://doi.org/10.1016/j.lwt.2021.111382&#160;<a class="footnote-backref" href="#fnref:3" title="Jump back to footnote 3 in the text">&#8617;</a></p>
</li>
</ol>
</div>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
    
    <nav class="md-footer__inner md-grid" aria-label="Footer" >
      
        
        <a href="../description/" class="md-footer__link md-footer__link--prev" aria-label="Previous: Description" rel="prev">
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
          </div>
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Previous
              </span>
              Description
            </div>
          </div>
        </a>
      
      
        
        <a href="../result/" class="md-footer__link md-footer__link--next" aria-label="Next: Result" rel="next">
          <div class="md-footer__title">
            <div class="md-ellipsis">
              <span class="md-footer__direction">
                Next
              </span>
              Result
            </div>
          </div>
          <div class="md-footer__button md-icon">
            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M4 11v2h12l-5.5 5.5 1.42 1.42L19.84 12l-7.92-7.92L10.5 5.5 16 11H4Z"/></svg>
          </div>
        </a>
      
    </nav>
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2022 iDEC. CC-BY-4.0 | The International Directed Evolution Competition (iDEC) is a Scottish Charitable Incorporate Organisation (SCIO), registered in Scotland, with charity number：SC050826
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.sections"], "search": "../assets/javascripts/workers/search.5bf1dace.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
    
    
      <script src="../assets/javascripts/bundle.078830c0.min.js"></script>
      
        <script src="../javascripts/config.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js"></script>
      
        <script src="../javascripts/tables.js"></script>
      
    
  </body>
</html>